src/SDK/Trace/Span.php

```diff
@@ -290,6 +292,18 @@ public function toSpanData(): SpanDataInterface
             $this->endEpochNanos,
             $this->hasEnded
         );
+
+        if ($spanData->getTotalDroppedLinks() || $spanData->getTotalDroppedEvents() || $spanData->getAttributes()->getDroppedAttributesCount()) {
```

Comment:
Should also check for dropped event or link attributes.

Reply:
Updated to check+log on LogRecord dropped attributes, span event+link dropped attributes.

Moved the check into span's `onEnd`, so that it is called once per span. This does add an extra call to `toSpanData`. I didn't profile the difference, but it looks like a computationally light function...